src/doc/src/commands/cargo-fetch.md

```diff
@@ -17,6 +17,10 @@ fetch` unless the lock file changes.
 If the lock file is not available, then this command will generate the lock
 file before fetching the dependencies.
 
+If the lock file is not compatible with the current `Cargo.toml`, then this
+command will initiate dependency resolution from the current `Cargo.toml` file,
+and update the lock file.
+
```

Comment:
https://doc.rust-lang.org/nightly/cargo/commands/cargo-fetch.html#option-cargo-fetch---locked

This is mentioned in `--locked` flag. Unsure if we need to emphasize it here.